src/webapp/azext_webapp/tunnel.py

```diff
@@ -109,7 +109,7 @@ def _listen(self):
         while True:
             self.client, _address = self.sock.accept()
             self.client.settimeout(1800)
-            host = 'wss://{}{}'.format(self.remote_addr, '.scm.azurewebsites.net/AppServiceTunnel/Tunnel.ashx')
+            host = '{}{}'.format(self.remote_addr.replace('https', 'wss'), '/AppServiceTunnel/Tunnel.ashx')
```

Comment:
add `if startwith http` check
and only replace `https` from the head of the string

Reply:
is it possible for kudu to run on http?